flow/flow.go

```diff
@@ -309,28 +312,66 @@ func (p *Packet) TransportFlow() (gopacket.Flow, error) {
 	return layer.TransportFlow(), nil
 }
 
+//Hash flow with custom func, ignore ports in transport level for VXLAN and Geneve
+func Hash(f gopacket.Flow, hasher hash.Hash, isTransportUDP bool) {
+	src, dst := f.Endpoints()
+	if isTransportUDP {
+		srcPort, _ := strconv.ParseInt(src.String(), 10, 64)
+		dstPort, _ := strconv.ParseInt(dst.String(), 10, 64)
+		if srcPort == 4789 || srcPort == 6081 {
```

Comment:
If you don't change the code line 368-369, you don't have to handle geneve nor vxlan here, IMHO

Reply:
maybe, but the whole reason I changed those lines was that the hash function that was being used wasn't working correctly, right?
one more thing though, if you take a look at the code of the hash function for this layer that was in use originally (which is in flows.go in gopacket's code) you won't see any special treatmen for those protocols, and yet the ports are always being hashed, which begs the question, how come this works?